_data/results.csv

```diff
@@ -594,3 +594,35 @@ Date,"Team Name","Team Size","Start Time","End Time"
 2024-06-11,Interstellar,2,09:31:00,
 2024-06-11,"4 Idiots",4,09:52:00,
 2024-06-11,"Pasta <3",2,10:07:00,
+2023-07-09,One is Company,1,07:31:00,08:37:00
```

Comment:
It's 2024 @karthikb351 😜

(broke in arcane ways because the code couldn't find this event, and there were no standings for the actual event.

Reply:
oh no, my bad =/